se.cendio.tlclient.json

```diff
@@ -0,0 +1,58 @@
+{
+	"app-id": "se.cendio.tlclient",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "21.08",
```

Comment:
The ThinLinc client has very mild system requirements, so perhaps this can be set so it always picks the latest?

Reply:
I'm not sure that's possible, and I'm not sure it's desired. There's always the risk that something stops working when you upgrade to the next runtime, even if it is unlikely. For that reason I think it's best to manually do this and test upgrades before you release them.